go-controller/pkg/ovn/obj_retry.go

```diff
@@ -640,6 +640,11 @@ func (oc *Controller) addResource(objectsToRetry *retryObjs, obj interface{}, fr
 			return err
 		}
 
+		if err = oc.updateEgressFirewallForNode(nil, node); err != nil {
```

Comment:
is it better to bundle the egfw node event with node resource or create new resource egressfwNode similar to `EgressNodeType` used by EIP ?

Reply:
hmm good point, let me try this

Reply:
done